src/mapgen.cpp

```diff
@@ -1082,15 +1085,14 @@ void MapgenParams::calcMapgenEdges()
 	// SAO position limits, in Irrlicht units
 	m_sao_limit_min = mapgen_edge_min * BS - 3.0f;
 	m_sao_limit_max = mapgen_edge_max * BS + 3.0f;
+	
+	m_sao_limit_calculated = true;
 }
 
 
 bool MapgenParams::saoPosOverLimit(const v3f &p)
 {
-	if (!m_sao_limit_calculated) {
-		calcMapgenEdges();
-		m_sao_limit_calculated = true;
-	}
+	calcMapgenEdges();
```

Comment:
Because `saoPosOverLimit(const v3f &p)` is called so often (every server step for every particle) i would rather not call another function to then return, i prefer that we keep this like this:
```
	if (!m_sao_limit_calculated)
		calcMapgenEdges();
```